Inaccessible links in Supabase blog

# Bug report

<!-- 

⚠️ We receive a lot of bug reports which have already been solved or discussed. If you are looking for help, please try these first:

- Docs: https://docs.supabase.com
- Discussions: https://github.com/supabase/supabase/discussions
- Discord: https://discord.supabase.com

Before opening a bug report, please verify the following: 

-->

- [x] I confirm this is a bug with Supabase, not with my own application. 
- [x] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Inaccessible links in Share article section of Supabase blog

Links for sharing articles at end of every post in Supabase blog do not provide a meaningful text to assistive technologys (screen readers). Some links provide text like share while others provide title of the post as text.

## To Reproduce

Steps to reproduce the behavior:

1. Go to any post in Supabase blog like [beta update august 2023](https://supabase.com/blog/beta-update-august-2023)
2. Scroll down to 'Share article section'
3. Red text of all sharing links with screen reader. Accessibilitytree which is given in Devtools also can be used to inspect text of sharing links.

## Expected behavior

Screen reader should announce name of the social media site where the article would be shared if clicked on that link. for example, share on X or Twitter.
## System information

- OS: Android, Windows
- Browser chrome
- screenreader: Talkback, NVDA,respectively.
## Want to contribute
If given an opportunity, I would like to fix the issue myself. Thank you so much.

@saltcod hi sir 
PS C:\Users\bayya\supabase> npm run dev

> supabase@0.0.0 dev
> turbo run dev --parallel

• Packages in scope: common, config, docs, shared-data, studio, supabase-tests, tsconfig, ui, www
• Running dev in 9 packages
• Remote caching disabled
docs:dev: cache bypass, force executing 821004325b9e6fd7
studio:dev: cache bypass, force executing 4c85edf186881cbb
www:dev: cache bypass, force executing dcb32f3bc942fd83
studio:dev: 
studio:dev: > studio@0.0.9 dev
studio:dev: > next dev -p 8082
studio:dev:
docs:dev: 
docs:dev: > docs@0.0.0 dev
docs:dev: > next dev --port 3001
docs:dev:
www:dev: 
www:dev: > www@0.0.3 dev
www:dev: > next --port 3000
www:dev:
docs:dev: node:internal/modules/esm/resolve:844
docs:dev:   throw new ERR_MODULE_NOT_FOUND(packageName, fileURLToPath(base), null);
docs:dev:         ^
docs:dev:
docs:dev: Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@next/mdx' imported from C:\Users\bayya\supabase\apps\docs\next.config.mjs
docs:dev:     at packageResolve (node:internal/modules/esm/resolve:844:9)
docs:dev:     at moduleResolve (node:internal/modules/esm/resolve:901:20)
docs:dev:     at defaultResolve (node:internal/modules/esm/resolve:1121:11)
docs:dev:     at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:396:12)
docs:dev:     at ModuleLoader.resolve (node:internal/modules/esm/loader:365:25)
docs:dev:     at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:240:38)
docs:dev:     at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)
docs:dev:     at link (node:internal/modules/esm/module_job:84:36) {
docs:dev:   code: 'ERR_MODULE_NOT_FOUND'
docs:dev: }
docs:dev:
docs:dev: Node.js v21.1.0
docs:dev: npm ERR! Lifecycle script `dev` failed with error:
docs:dev: npm ERR! Error: command failed 
docs:dev: npm ERR!   in workspace: docs@0.0.0
docs:dev: npm ERR!   at location: C:\Users\bayya\supabase\apps\docs
studio:dev: node:internal/modules/cjs/loader:1144
studio:dev:   const err = new Error(message);
studio:dev:               ^
studio:dev:
studio:dev: Error: Cannot find module '@sentry/nextjs'
studio:dev: Require stack:
studio:dev: - C:\Users\bayya\supabase\studio\next.config.js
studio:dev:     at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
studio:dev:     at C:\Users\bayya\AppData\Roaming\npm\node_modules\next\dist\server\require-hook.js:54:36
studio:dev:     at Module._load (node:internal/modules/cjs/loader:985:27)
studio:dev:     at Module.require (node:internal/modules/cjs/loader:1235:19)
studio:dev:     at mod.require (C:\Users\bayya\AppData\Roaming\npm\node_modules\next\dist\server\require-hook.js:64:28)
studio:dev:     at require (node:internal/modules/helpers:176:18)
studio:dev:     at Object.<anonymous> (C:\Users\bayya\supabase\studio\next.config.js:1:30)
studio:dev:     at Module._compile (node:internal/modules/cjs/loader:1376:14)
studio:dev:     at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
studio:dev:     at Module.load (node:internal/modules/cjs/loader:1207:32) {
studio:dev:   code: 'MODULE_NOT_FOUND',
studio:dev:   requireStack: [ 'C:\\Users\\bayya\\supabase\\studio\\next.config.js' ]
studio:dev: }
studio:dev:
studio:dev: Node.js v21.1.0
docs:dev: ERROR: command finished with error: command (C:\Users\bayya\supabase\apps\docs) npm run dev exited (1)     
studio:dev: npm ERR! Lifecycle script `dev` failed with error:
studio:dev: npm ERR! Error: command failed
studio:dev: npm ERR!   in workspace: studio@0.0.9
studio:dev: npm ERR!   at location: C:\Users\bayya\supabase\studio
www:dev: node:internal/modules/esm/resolve:844
www:dev:   throw new ERR_MODULE_NOT_FOUND(packageName, fileURLToPath(base), null);
www:dev:         ^
www:dev: 
www:dev: Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@next/bundle-analyzer' imported from C:\Users\bayya\supabase\apps\www\next.config.mjs
www:dev:     at packageResolve (node:internal/modules/esm/resolve:844:9)
www:dev:     at moduleResolve (node:internal/modules/esm/resolve:901:20)
www:dev:     at defaultResolve (node:internal/modules/esm/resolve:1121:11)
www:dev:     at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:396:12)
www:dev:     at ModuleLoader.resolve (node:internal/modules/esm/loader:365:25)
www:dev:     at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:240:38)
www:dev:     at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)
www:dev:     at link (node:internal/modules/esm/module_job:84:36) {
www:dev:   code: 'ERR_MODULE_NOT_FOUND'
www:dev: }
www:dev:
www:dev: Node.js v21.1.0
www:dev: npm ERR! Lifecycle script `dev` failed with error: 
www:dev: npm ERR! Error: command failed
www:dev: npm ERR!   in workspace: www@0.0.3
www:dev: npm ERR!   at location: C:\Users\bayya\supabase\apps\www
docs#dev: command (C:\Users\bayya\supabase\apps\docs) npm run dev exited (1)

 Tasks:    0 successful, 3 total
Cached:    0 cached, 3 total
  Time:    2.885s
Failed:    docs#dev

 ERROR  run failed: command  exited (1)

Sir if u don't mind could u please help out...i wanna contribute in supabase and I'm a beginner

--- a/apps/www/components/Blog/ShareArticleActions.tsx
+++ b/apps/www/components/Blog/ShareArticleActions.tsx
@@ -4,7 +4,7 @@ import { IconLinkedinSolid, IconTwitterX, IconYCombinator } from 'ui'
 const ShareArticleActions = ({ title, slug }: { title: string; slug: string }) => (
   <div className="mt-4 flex items-center space-x-4">
     <Link
-        aria-label="Share on X "
+      aria-label="Share on X"
       href={`https://twitter.com/share?text=${title}&url=https://supabase.com/blog/${slug}`}
       target="_blank"
       className="text-foreground-lighter hover:text-foreground"
@@ -13,7 +13,7 @@ const ShareArticleActions = ({ title, slug }: { title: string; slug: string }) =
     </Link>
 
     <Link
-        aria-label="Share on Linkedin "
+      aria-label="Share on Linkedin"
       href={`https://www.linkedin.com/shareArticle?url=https://supabase.com/blog/${slug}&title=${title}`}
       target="_blank"
       className="text-foreground-lighter hover:text-foreground"
@@ -21,7 +21,7 @@ const ShareArticleActions = ({ title, slug }: { title: string; slug: string }) =
       <IconLinkedinSolid size={24} />
     </Link>
     <Link
-        aria-label="Share on Newycombinator "
+      aria-label="Share on Hacker News"
       href={`https://news.ycombinator.com/submitlink?u=https://supabase.com/blog/${slug}&t=${title}`}
       target="_blank"
       className="text-foreground-lighter hover:text-foreground"

--- a/apps/www/components/Blog/ShareArticleActions.tsx
+++ b/apps/www/components/Blog/ShareArticleActions.tsx
@@ -4,6 +4,7 @@ import { IconLinkedinSolid, IconTwitterX, IconYCombinator } from 'ui'
 const ShareArticleActions = ({ title, slug }: { title: string; slug: string }) => (
   <div className="mt-4 flex items-center space-x-4">
     <Link
+        aria-label="Share on X "
       href={`https://twitter.com/share?text=${title}&url=https://supabase.com/blog/${slug}`}
       target="_blank"
       className="text-foreground-lighter hover:text-foreground"
@@ -12,6 +13,7 @@ const ShareArticleActions = ({ title, slug }: { title: string; slug: string }) =
     </Link>
 
     <Link
+        aria-label="Share on Linkedin "
       href={`https://www.linkedin.com/shareArticle?url=https://supabase.com/blog/${slug}&title=${title}`}
       target="_blank"
       className="text-foreground-lighter hover:text-foreground"
@@ -19,6 +21,7 @@ const ShareArticleActions = ({ title, slug }: { title: string; slug: string }) =
       <IconLinkedinSolid size={24} />
     </Link>
     <Link
+        aria-label="Share on Newycombinator "
       href={`https://news.ycombinator.com/submitlink?u=https://supabase.com/blog/${slug}&t=${title}`}
       target="_blank"
       className="text-foreground-lighter hover:text-foreground"

